cli/app.go

```diff
@@ -1144,6 +1147,45 @@ var app = &cli.App{
 							},
 							Action: RobotsPartShellAction,
 						},
+						{
+							Name:        "cp",
+							Usage:       "copy files to and from a machine part",
+							Description: `In order to use the cp command, the machine must have a valid shell type service.`,
+							UsageText: createUsageText(
+								"machines part cp",
+								[]string{organizationFlag, locationFlag, machineFlag, partFlag},
+								true,
+								"[-p] [-r] source ([machine:]files) ... target ([machine:]files"),
+							Flags: []cli.Flag{
+								&cli.StringFlag{
+									Name: organizationFlag,
```

Comment:
```suggestion
									Name: organizationFlag,
									Required: true,
```
`organizationFlag`, `locationFlag`, `machineFlag`, and `partFlag` are all documented as required for this command but not in the code.

Reply:
based on the logic of how the selects work, it's not required like other commands. the CLI auto selects orgs and other arguments based on whats in the user's account